atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/KvsDeleteBenchmarks.java

```diff
@@ -50,4 +65,18 @@ public Object batchDelete(RegeneratingTable.KvsBatchRegeneratingTable table) {
         return doDelete(table);
     }
 
+    @Benchmark
+    @Warmup(time = 1, timeUnit = TimeUnit.SECONDS)
```

Comment:
Do you have stats on how many iterations you got through during warmup and measurement?